kernel/packages/scene-system/stateful.scene.system.ts

```diff
@@ -80,6 +80,16 @@ class StatefulWebWorkerScene extends Script {
           .catch((error) => this.error(`Failed to save the scene's manifest`, error))
       }
     })
+
+    // Listen to save scene info requests
+    this.eventSubscriber.on('stateEvent', ({ data }) => {
+      const { type, payload } = data
+      if (type === 'SaveSceneInfo') {
+        this.sceneStateStorage
+          .saveSceneInfo(serializeSceneState(this.sceneDefinition), payload.title, payload.description, payload.screenshot)
+          .catch((error) => this.error(`Failed to save the scene's info`, error))
+      }
+    })
```

Comment:
we already have the `SaveSceneState` event type. why not using that instead of creating a new one?

Reply:
`saveSceneState` and `saveSceneInfo` do different things and are called in different situations:
- `saveSceneState`: updates the entities states of the project manifest and is called each time a change is done in the BiW editor.
- `saveSceneInfo`: updates the basic info of the project manifest (name, description and screenshot) and is called only in 3 situation: when we create a new project, when we exit BiW editor and there have been changes or when we publish.

Reply:
Yes, it seems a more descriptive name 👍

Reply:
Rememeber to change it in the unity part too!

Reply:
`SaveProjectInfo` renaming donde in https://github.com/decentraland/explorer/pull/2432/commits/7b545b17ce9ad8bcadccad730134e2036f869555.

Reply:
`SaveProjectInfo` renaming (Unity side) done in: https://github.com/decentraland/unity-renderer/pull/628/commits/858e0ff990730807d69ab4d294cac6a666adf3c6